webapp/src/components/MeetingQuickView.tsx

```diff
@@ -0,0 +1,75 @@
+import { Stack, Typography } from '@mui/material';
+import React, { useContext, useState } from 'react';
+
+import { EntityId, Meeting as APIMeeting } from '../types/api';
+import CreateMeetingNoteForm from './CreateMeetingNoteForm';
+import SessionContext from './SessionContext';
+import useApiData from '../helpers/useApiData';
+
+interface MeetingQuickViewProps {
+  meetingId?: EntityId;
```

Comment:
Passing down `meetingId` is optional here but I don't think it should be optional.

Reply:
I've removed it.